guide/popular-topics/reloading-commands.md

```diff
@@ -0,0 +1,62 @@
+# Reloading commands
+
+When writing your commands, you may find it tedious to restart your bot every time for testing the smallest changes, With a command handler, you can eliminate this issue and reload your commands while your bot is running.
+
+This section assumes you did follow the [Command Handling](/guide/creating-your-bot/command-handling.md) part.
+
+::: warning
+The require method does not work in ESM. If you are using it, try out the [hot-esm](https://www.npmjs.com/package/hot-esm) module.
+:::
```

Comment:
```suggestion
::: warning
ESM does not support require and clearing import cache, you can use [hot-esm](https://www.npmjs.com/package/hot-esm) to import files without cache.
:::
```
esm users would already know about require not working so it's better to give a heads up that import cache is not manageable either